Dikke Ploaten/Services/Database.swift

```diff
@@ -39,12 +42,12 @@ class Database {
 			} else {
 				print("Document added with ID")
 			}
-			failureHandler(err)
+			completionHandler(err)
 		}
 	}
 	
 	// Updates user's collection when album gets added/deleted
-	func updateCollection(albums: [Album], completionHandler: @escaping (_ updatedCollection: [Album]) -> ()) {
+	func getCollection(albums: [Album], completionHandler: @escaping (_ updatedCollection: [Album]) -> ()) {
```

Comment:
Hmm lijkt me een vreemde methodenaam, misschien is iets zoals `addCollectionChangeListener` beter?

Reply:
Ahn dacht dat jij zei dat het beter 'get' was ipv 'update'
<img width="621" alt="Screenshot 2019-03-28 at 14 20 36" src="https://user-images.githubusercontent.com/25817707/55160921-acd91c80-5164-11e9-8593-ac6885a53d2c.png">